service/controller/clusterapi/v30/adapter/spec.go

```diff
@@ -45,10 +45,10 @@ const (
 	httpsPort = 443
 )
 
-// APIWhitelist defines guest cluster k8s api whitelisting.
+// APIWhitelist defines guest cluster k8s public/private api whitelisting.
 type APIWhitelist struct {
-	Enabled    bool
-	SubnetList string
+	Private Whitelist
+	Public  Whitelist
```

Comment:
Don't we need to update CF templates now?

Reply:
The template didn't change. It just receives rules, which are processed before passing into template